apps/portal-api/src/modules/services/document/document.domain.ts

```diff
@@ -898,3 +853,21 @@ export const loadImagesByDocumentId = async (documentId: string) => {
   }
   return images;
 };
+
+const addIncludeMetadataQuery = (
+  qb: Knex.QueryBuilder,
+  include_metadata: string[]
+) => {
+  (include_metadata ?? []).forEach((metaKey, index) => {
```

Comment:
Look a bit cleaner IMO to define a default params instead.
```suggestion
const addIncludeMetadataQuery = (
  qb: Knex.QueryBuilder,
  include_metadata: string[] = []
) => {
  (include_metadata).forEach((metaKey, index) => {
```